### ISSUE TITLE
disable->enable->join errors

### ISSUE THREAD

**Issue by mooreatv** [mooreatv/DynamicBoxer#54]
```
3x DynamicBoxer\DynamicBoxer-@project-version@.lua:109: bad argument #1 to 'ipairs' (table expected, got nil)
[C]: in function `ipairs'
DynamicBoxer\DynamicBoxer-@project-version@.lua:109: in function `Replace'
DynamicBoxer\DynamicBoxer-@project-version@.lua:158: in function `SetMacro'
ISBoxer\ISBoxer_Character.lua:7: in function `LoadBinds'
ISBoxer\ISBoxer-1.0.lua:153: in function `LoadBinds'
DynamicBoxer\DynamicBoxer-@project-version@.lua:152: in function `LoadBinds'
ISBoxer\ISBoxer-1.0.lua:178: in function <ISBoxer\ISBoxer.lua:173>

Locals:
(*temporary) = nil
(*temporary) = "table expected, got nil"
 = <function> defined =[C]:-1


7x DynamicBoxer\DynamicBoxer-@project-version@.lua:919: attempt to index field 'ISBTeam' (a nil value)
DynamicBoxer\DynamicBoxer-@project-version@.lua:919: in function `ProcessMessage'
DynamicBoxer\DynamicBoxer-@project-version@.lua:999: in function <DynamicBoxer\DynamicBoxer.lua:994>
(tail call): ?

Locals:
self = DynBoxer {
 WatchedTable = <function> defined @DynamicBoxer\MoLib/MoLib.lua:662
 joinDone = true
 VerifySecureMessage = <function> defined @DynamicBoxer\MoLib/MoLib.lua:429
 RealmAbbrev = <function> defined @DynamicBoxer\MoLib/MoLib.lua:235
 SlotCommand = <function> defined @DynamicBoxer\DynamicBoxer.lua:482
 debugPrint = <function> defined @DynamicBoxer\MoLib/MoLib.lua:113
 Help = <function> defined @DynamicBoxer\DynamicBoxer.lua:1286
 randomIdLen = 8
 DebugEvCall = <function> defined @DynamicBoxer\MoLib/MoLib.lua:301
 firstMsg = 1
 InitRealms = <function> defined @DynamicBoxer\MoLib/MoLib.lua:164
 manual = 3
 Warning = <function> defined @DynamicBoxer\MoLib/MoLib.lua:128
 secureFutureThreshold = -5
 manualTeamSize = 5
 refresh = 2
 securePastThreshold = 45
 currentCount = 0
 ManualSetup = <function> defined @DynamicBoxer\DynamicBoxer.lua:230
 InfoPayload = <function> defined @DynamicBoxer\DynamicBoxer.lua:478
 ChatFilter = <function> defined @DynamicBoxer\DBoxChat.lua:15
 maxIter = 1
 extractRealmID = <function> defined @DynamicBoxer\MoLib/MoLib.lua:224
 Disband = <function> defined @DynamicBoxer\DynamicBoxer.lua:769
 autoInviteSlot = 1
 debugLog = <table> {
 }
 GetMyFQN = <function> defined @DynamicBoxer\MoLib/MoLib.lua:306
 optionsPanel = <unnamed> {
 }
 MoLibInstallInto = <function> defined @DynamicBoxer\MoLib/MoLib.lua:57
 OnUICancel = <function> defined @DynamicBoxer\DBoxUI.lua:80
 ReplaceAll = <function> defined @DynamicBoxer\MoLib/MoLib.lua:472
 SameRealmAsUs = <function> defined @DynamicBoxer\DynamicBoxer.lua:407
 EMA = <table> {
 }
 NumToHex = <table> {
 }
 roundUp = <function> defined @DynamicBoxer\MoLib/MoLibUI.lua:19
 SendDirectMessage = <function> defined @DynamicBoxer\DynamicBoxer.lua:439
 DumpT = <table> {
 }
 EventD = <table> {
 }
 syncNum = 1
 statusPos = <table> {
 }
 ShortHash = <function> defined @DynamicBoxer\MoLib/MoLib.lua:372
 Replace = <function> defined @DynamicBoxer\DynamicBoxer.lua:104
 round = <function> defined @DynamicBoxer\MoLib/MoLibUI.lua:13
 GetRealRealm = <function> defined @DynamicBoxer\MoLib/MoLib.lua:208
 deepmerge = <function> defined @DynamicBoxer\MoLib/MoLib.lua:24
 GetLocalization = <function> defined @DynamicBoxer\MoLib/MoLib.lua:641
 Dump = <function> defined @DynamicBoxer\MoLib/MoLib.lua:288
 name = "DynamicBoxer"
 InPartyWith = <function> defined @DynamicBoxer\DynamicBoxer.lua:430
 manifestVersion = "vX.Y.Z"
 whisperPrefix = "DynamicBoxer~"
 MoLibInit = <function> defined @DynamicBoxer\MoLib/MoLib.lua:152
 Secret = "pMc6nT7o"
 SnapFrame = <function> defined @DynamicBoxer\MoLib/MoLibUI.lua:34
 Team = <table> {
 }
 L = <table> {
 }
 AddToMembersHistory = <function> defined @DynamicBoxer\DynamicBoxer.lua:637
 Debug = <function> defined @DynamicBoxer\MoLib/MoLib.lua:98
 NewestCrossRealmMaster = <function> defined @DynamicBoxer\DynamicBoxer.lua:346
 MasterName = "Áßçðéöýøîÿæü-Blade'sEdge"
 IsActive = <function> defined @DynamicBoxer\DynamicBoxer.lua:96
 SetupStatusUI = <function> defined @DynamicBoxer\DBoxUI.lua:699
 watched = <table> {
 }
 autoInvite = false
 PartyToggle = <function> defined @DynamicBoxer\DynamicBoxer.lua:722
 teamHistory = <table> {
 }
 shortName = "Tabbu"
 format = <function> defined @DynamicBoxer\MoLib/MoLib.lua:74
 enabled = false
 GetFactionTexture = <function> defined @DynamicBoxer\DBoxUI.lua:637
 inUI = false
 tokenMinLen = 22
 expectedCount = 0
 PrintInfo = <function> defined @Inter
```

**Comment by mooreatv** [comment]
also
```
6x DynamicBoxer\DynamicBoxer-@project-version@.lua:109: bad argument #1 to 'ipairs' (table expected, got nil)
[C]: in function `ipairs'
DynamicBoxer\DynamicBoxer-@project-version@.lua:109: in function `Replace'
DynamicBoxer\DynamicBoxer-@project-version@.lua:158: in function `SetMacro'
ISBoxer\ISBoxer_Character.lua:7: in function `LoadBinds'
ISBoxer\ISBoxer-1.0.lua:153: in function `LoadBinds'
DynamicBoxer\DynamicBoxer-@project-version@.lua:152: in function `LoadBinds'
ISBoxer\ISBoxer-1.0.lua:178: in function <ISBoxer\ISBoxer.lua:173>

Locals:
(*temporary) = nil
(*temporary) = "table expected, got nil"
 = <function> defined =[C]:-1
```

**Comment by mooreatv** [comment]
fixed one of them (was a leftover self.enabled pre rename to .watched.enabled)

**Comment by mooreatv** [comment]
fixed in 1.9.6
